src/components/data-exploration/embedding/Embedding.jsx

```diff
@@ -32,10 +32,9 @@ import {
 } from 'utils/plotUtils';
 import getContainingCellSetsProperties from 'utils/cellSets/getContainingCellSetsProperties';
 
-const Scatterplot = dynamic(
-  () => import('vitessce/dist/umd/production/scatterplot.min').then((mod) => mod.Scatterplot),
```

Comment:
I think if you use the relative path  it will work. Can you try?

Reply:
If it works can you do it in the other places (I saw the same for the heatmap import)

Reply:
I tried that and it doesn't work.

Reply:
Then I recommend you put all the vitessce wrappers in one file so its less confusing

Reply:
Good idea! I have done that in my new branch and PR.